drivers/ethernet/eth.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2020 Linaro Ltd
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_ETHERNET_ETH_H_
+#define ZEPHYR_DRIVERS_ETHERNET_ETH_H_
+
+#include <zephyr/types.h>
+
+static inline void gen_random_mac(u8_t *mac_addr, u8_t b0, u8_t b1, u8_t b2)
```

Comment:
No option for a random OUI?  I guess nobody uses one.

Reply:
I based it on what the drivers did to date.